rules/aip0203/field_behavior_required.go

```diff
@@ -23,47 +23,62 @@ import (
 	"github.com/jhump/protoreflect/desc"
 )
 
-var minimumRequiredFieldBehavior = stringset.New(
+var fbs = stringset.New(
 	"OPTIONAL", "REQUIRED", "OUTPUT_ONLY", "IMMUTABLE",
 )
 
 var fieldBehaviorRequired = &lint.MethodRule{
 	Name: lint.NewRuleName(203, "field-behavior-required"),
 	LintMethod: func(m *desc.MethodDescriptor) []lint.Problem {
-		// we only check requests, as OutputTypes are always
-		// OUTPUT_ONLY
-		it := m.GetInputType()
-		return checkFields(it)
+		req := m.GetInputType()
+		ps := problems(req)
+		if len(ps) == 0 {
```

Comment:
this could be returned directly too: 0 problems wouldn't add any.

Reply:
All the other rules return nil for no problems.